conf/modules.config

```diff
@@ -64,6 +64,12 @@ params {
         }
         'prokka' {
             args                = ''
+            publish_files       = false
+        }
+        'cat_cat' {
```

Comment:
Perhaps this should have another name, one that is more transparent and shows that it's called for Prokka output -- `prokka_cat` maybe?

Reply:
Yes, seems logical! At first I thought the name needed to be the same as the actual module, but understood later that it wasn't the case.